pandas/core/reshape/merge.py

```diff
@@ -1635,7 +1636,11 @@ def _get_merge_keys(self):
                 )
             )
 
-            if is_datetime64_dtype(lt) or is_datetime64tz_dtype(lt):
+            if(
```

Comment:
This may throw a linter warning (space after the `if`).

Reply:
black wants to put a space here